content/en/docs/components/serving/kfserving.md

```diff
@@ -26,7 +22,9 @@ Our strong community contributions help KFServing to grow. We have a Technical S
 
 ## Install with Kubeflow
 KFServing works with Kubeflow 1.1. Kustomize installation files are [located in the manifests repo](https://github.com/kubeflow/manifests/tree/master/kfserving).
-See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex) and [GCP](https://github.com/kubeflow/kfserving/tree/master/docs/samples/gcp-iap)
+See examples running KFServing on [Istio/Dex](https://github.com/kubeflow/kfserving/tree/master/docs/samples/istio-dex). For installation on major cloud providers with Kubeflow, please follow their installation docs. 
```

Comment:
are there kfserving specific docs from the cloud providers?

Reply:
not kfserving specific - but when you deploy with Kubeflow.